src/plugins/layout/shadowdom.ts

```diff
@@ -1,10 +1,9 @@
 import { assert, isNumber, traverseNodeTree } from "../../utils";
-import { getNodeIndex, IgnoreTag, NodeIndex } from "./stateprovider";
+import { getNodeIndex, NodeIndex, shouldIgnoreNode } from "./stateprovider";
 
 // Class names for child list mutation classifications
 const FinalClassName = "cl-final";
 const NewNodeClassName = "cl-new";
-const TopNewNodeClassName = "cl-new-top";
 const MovedNodeClassName = "cl-moved";
 const UpdatedNodeClassName = "cl-updated";
```

Comment:
Based on the code below it seemed like "Moved" represented "Deleted". If that's correct, rename to reflect that.